client/src/main/java/com/sap/mobile/services/client/push/ApnsNotification.java

```diff
@@ -480,6 +497,7 @@ private static class ApnsNotificationObject implements ApnsNotification {
 			private final List<String> urlArguments;
 			private final String threadId;
 			private final Boolean mutableContent;
+			private final InterruptionLevel interruptionLevel;
```

Comment:
If I am not mistaken, the property must be exposed in the [DTOApnsNotification](https://github.com/SAP/java-integration-for-sap-mobile-services/blob/main/client/src/main/java/com/sap/mobile/services/client/push/DTO.java#L70) class in order to be sent to Mobile Services.

Reply:
Done